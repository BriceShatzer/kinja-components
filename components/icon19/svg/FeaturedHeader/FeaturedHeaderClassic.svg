<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" aria-label="FeaturedHeaderClassic icon" viewBox="0 0 18 18">
	<path fill-rule="evenodd" d="M16 8a1 1 0 0 1 1 1v7a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V9a1 1 0 0 1 1-1h14zm0 1H2v7h14V9zm-2.5-5a.5.5 0 1 1 0 1h-9a.5.5 0 0 1 0-1zm3-3a.5.5 0 1 1 0 1h-15a.5.5 0 0 1 0-1z"/>
</svg>
