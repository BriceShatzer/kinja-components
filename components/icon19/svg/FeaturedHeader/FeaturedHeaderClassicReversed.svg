<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" aria-label="FeaturedHeaderClassicReversed icon" viewBox="0 0 18 18">
	<path fill-rule="evenodd" d="M13.5 16a.5.5 0 0 1 .09 1H4.5a.5.5 0 0 1-.09-1h9.09zm3-3a.5.5 0 0 1 .09 1H1.5a.5.5 0 0 1-.09-1H16.5zM16 1a1 1 0 0 1 1 1v7a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1zm0 1H2v7h14V2z"/>
</svg>
