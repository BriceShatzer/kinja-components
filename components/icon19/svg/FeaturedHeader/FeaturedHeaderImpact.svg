<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" aria-label="FeaturedHeaderImpact icon" viewBox="0 0 18 18">
	<path fill-rule="evenodd" d="M16 1a1 1 0 0 1 1 1v14a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1h14zm0 1H2v14h14V2zM9.5 12a.5.5 0 1 1 0 1h-5a.5.5 0 1 1 0-1zm2-3a.5.5 0 1 1 0 1h-7a.5.5 0 1 1 0-1z"/>
</svg>
