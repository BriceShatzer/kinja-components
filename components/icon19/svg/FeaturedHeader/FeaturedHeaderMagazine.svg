<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" aria-label="FeaturedHeaderMagazine icon" viewBox="0 0 18 18">
	<path fill-rule="evenodd" d="M17 4a1 1 0 0 1 1 1.01v7.98A1 1 0 0 1 17 14H7a1 1 0 0 1-1-1.01V5.01A1 1 0 0 1 7 4h10zM2.5 13a.5.5 0 1 1 0 1h-2a.5.5 0 1 1 0-1zM17 5H7v8h10V5zM3.5 10a.5.5 0 1 1 0 1h-3a.5.5 0 1 1 0-1zm-1-3a.5.5 0 0 1 0 1h-2a.5.5 0 0 1 0-1zm1-3a.5.5 0 0 1 0 1h-3a.5.5 0 0 1 0-1z"/>
</svg>
