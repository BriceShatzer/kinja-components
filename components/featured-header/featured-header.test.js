import * as React from 'react';
import { shallow } from 'enzyme';

import FeaturedHeader from './featured-header';
const featuredMediaImageOptions = {
	id: 'tscqkiavefq37idpvl5g',
	format: 'png',
	width: 1920,
	height: 1080,
	alignment: 'Bleed',
	caption: [],
	syndicationRights: false,
	attribution: [],
	type: 'Image'
};
describe('<ImpactHeader />', () => {
	it('should render by default', () => {
		const wrapper = shallow(
			<FeaturedHeader
				featuredMedia={featuredMediaImageOptions}
				title={'Featured Header'}
			/>
		);
		expect(wrapper).toMatchSnapshot();
	});
});
