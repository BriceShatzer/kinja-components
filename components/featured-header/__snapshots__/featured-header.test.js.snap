// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImpactHeader /> should render by default 1`] = `
<EnsureDefaultTheme>
  <header-wrapper__HeaderWrapper>
    <featured-header__FeaturedMediaWrapper>
      <featured-header__FeaturedMediaImageWrapper>
        <featured-header__GenericImageWrapper>
          <WithPlatform
            format="png"
            id="tscqkiavefq37idpvl5g"
            noLazy={true}
            relative={true}
            transform={null}
          />
        </featured-header__GenericImageWrapper>
      </featured-header__FeaturedMediaImageWrapper>
    </featured-header__FeaturedMediaWrapper>
    <featured-header__FeaturedTitleWrapper>
      <featured-header__TitleTag>
        <Link
          dangerouslySetInnerHTML={
            Object {
              "__html": "Featured Header",
            }
          }
          events={
            Array [
              Array [
                "Permalink page click",
                "Permalink page click - post header",
                "standard",
              ],
            ]
          }
        />
      </featured-header__TitleTag>
    </featured-header__FeaturedTitleWrapper>
  </header-wrapper__HeaderWrapper>
</EnsureDefaultTheme>
`;
